src/cellEditors/ComboBox.js

```diff
@@ -139,7 +143,7 @@ var ComboBox = Textfield.extend('ComboBox', {
 
             setModeIconAndOptgroup.call(this, toggle, mode.name, menuModes[mode.name]);
         }.bind(this));
-
+        this.grid.properties.scrollingEnabled = false;
```

Comment:
This line doesn't seem to effect scrolling at all. Anyway I don't think we need to suppress scrolling at this point. The only reason we were entertaining that change was to fix the issue, but you've already fixed it.